gui/src/components/Notifications.js

```diff
@@ -50,7 +51,12 @@ export class Notifications extends React.Component {
         }
         {successes.length > 0 && (
           <Flash success className={classes.flash}>
-            {successes.map((succ, i) => this.successPresenter(succ, i))}
+            {successes.map((succ, i) => {
+              if (succ.type === 'component') {
+                return <p key={i}>{succ.component(succ.props)}</p>
```

Comment:
Why not put this in the `successPresenter`? I would lean towards reducing ifs in JSX.

Reply:
@dimroc I feel like we should locate the notification structure in the same place that controls their creation. In this case the `Edit` & `New` bridge components. This branch is intended to be temporary until we use this pattern on the old components. 

If we continue with the current pattern the `Notifications` component will need to know about every possible notification type.